### ISSUE TITLE
[gff_harari] Missing pdf files in help folder

### ISSUE THREAD

**Issue by darcywong00** [keymanapp/keyboards#2209]
From #2196 and #2206

CI link checker failed in keymanapp/help.keyman.com#710

@dyacob 
The following two help files are missing from /release/gff/gff_harari/source/help/
* HarariTyping-English.pdf (copy from source/
* HarariTyping-Harari.pdf - This seems to be missing in the repo

**Comment by dyacob** [comment]
Hi @darcywong00 , I see links to these files coming from the `.php` page.  I've made a PR to fix this.  The HarariTyping-Harari.pdf link was removed since it doesn't exist.  I also fixed a name issue that I noticed in the `$pagename` and `$pagetitle` variables.

**Comment by mcdurdin** [comment]
Reopening because there is another link to HarariTyping-Latin.pdf on L208-L210:

```html
See <a target="_blank" href='HarariTyping-English.pdf'>Typing in Harari (English)</a> or 
<a target="_blank" href='HarariTyping-Harari.pdf'>Typing in Harari (Harari)</a> 
for full details on how to type all Harari letters, numbers and punctuation.
```

**Comment by mcdurdin** [comment]
Resolved in #2244.
